### PR TITLE
Support playing youtube videos as background

### PR DESCRIPTION
**Pull request by jtsymon**

### CODE REVIEW

**Comment by jtsymon** [comment]
For some reason some videos don't work, I think its an OpenCV problem because Pafy finds the video URL and then it fails to load. Most videos work though, now I just need to make it support changing video without restarting the script and dropping the webcam for a few seconds.

**Comment by likeablob** [comment]
@jtsymon 
Thank you for the nice PR!  
Yeah, by specifying URL directly we can avoid the extra downloading step.  Let me review this.

File: app.py [hunk 1]
```diff
@@ -279,6 +284,9 @@ def main(ARGS):
     parser.add_argument('-e', '--bg-video', metavar='VIDEO_FILE',
                         type=str, default=None,
                         help='Background video.')
+    parser.add_argument('-u', '--bg-url', metavar='VIDEO_FILE',
```
Comment (likeablob) [inline app.py]: Could you also change `metavar` to 'YOUTUBE_URL' or something like that?
```suggestion
    parser.add_argument('-u', '--bg-url', metavar='YOUTUBE_URL,
```

**Review by likeablob — CHANGES_REQUESTED** [review]
Almost LGTM. Could you please check the comment? :pray:

**Comment by likeablob** [comment]
@jtsymon Thanks :+1: ! I'd like to merge this PR.

BTW, Are you being forced to use a "secured" laptop too?

**Comment by jtsymon** [comment]
No, I'm just running Linux and liked this better than https://github.com/fangfufu/Linux-Fake-Background-Webcam which uses NodeJS as well as Python :slightly_smiling_face:

**Comment by likeablob** [comment]
@jtsymon Aha, that's nice to hear haha.

> now I just need to make it support changing video without restarting the script and dropping the webcam for a few seconds.

Regarding this matter, I'm not 100% sure but,  [`akvcam`](https://github.com/webcamoid/akvcam) could be used as an alternative to `v4l2loopback`, Since the former has the default picture feature.

> Configurable default picture in case no input signal available.

**Pull request merged**